FWCore/Framework/interface/LuminosityBlock.h

```diff
@@ -136,6 +136,8 @@ namespace edm {
 
     Provenance getProvenance(BranchID const& theID) const;
 
+    StableProvenance getStableProvenance(BranchID const& theID) const;
```

Comment:
Should this be 
```suggestion
    StableProvenance const& getStableProvenance(BranchID const& theID) const;
```
as in `Event.h`?

Reply:
done